smart-contract-verifier/src/compiler/compilers.rs

```diff
@@ -141,7 +149,8 @@ mod tests {
                 let fetcher = ListFetcher::new(url, temp_dir(), None, None)
                     .await
                     .expect("Fetch releases");
-                let compilers = Compilers::new(Arc::new(fetcher), SolidityCompiler::new());
+                let lock = Arc::new(Semaphore::new(1));
```

Comment:
I think we can allow some multithreading in tests too. may be make it default (threads count) or just a larger number (2-4)?